language/unit_tests/test_client.py

```diff
@@ -17,28 +17,33 @@
 
 class TestClient(unittest.TestCase):
 
-    def _getTargetClass(self):
+    @staticmethod
+    def _get_target_class():
```

Comment:
If we're going down this path (of debatable value), then it would seem better to just do it via a script across the whole repo.  Same for the `_makeOne` -> `_make_one` change below, and I assume `_callFUT` -> `_call_fut` where appropriate.